services/web/server/src/simcore_service_webserver/users/_notifications.py

```diff
@@ -26,8 +26,10 @@ class NotificationCategory(StrAutoEnum):
 
 class BaseUserNotification(BaseModel):
     user_id: UserID
+    user_from_id: Literal[None] | UserID = None
     category: NotificationCategory
     actionable_path: str
+    resource_id: Literal[""] | str = ""
     title: str
```

Comment:
are these not always filled up?

Reply:
Nope, not in already existing notifications. The new ones will contain that information, but I still need to be able to read the old ones.

I want to I default them to those values. Is that correct?